babybear/src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,126 @@
+
+    public static final int FRONT_LEFT_MODULE_DRIVE_MOTOR = 1;
+    public static final int FRONT_LEFT_MODULE_STEER_MOTOR = 2;
+    public static final int FRONT_LEFT_MODULE_STEER_ENCODER = 3;
+    public static final double FRONT_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(147.92 + 344.26 + 307.26);
+
+    public static final int FRONT_RIGHT_MODULE_DRIVE_MOTOR = 4;
+    public static final int FRONT_RIGHT_MODULE_STEER_MOTOR = 5;
+    public static final int FRONT_RIGHT_MODULE_STEER_ENCODER = 6;
+    public static final double FRONT_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(108.15 + 282.74 + 0.264 + .9);
+
+    public static final int BACK_LEFT_MODULE_DRIVE_MOTOR = 7;
+    public static final int BACK_LEFT_MODULE_STEER_MOTOR = 8;
+    public static final int BACK_LEFT_MODULE_STEER_ENCODER = 9;
+    public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(77.639 + 259.80 + 180.0 + 348.71 + 8.0);
+
+    public static final int BACK_RIGHT_MODULE_DRIVE_MOTOR = 10;
+    public static final int BACK_RIGHT_MODULE_STEER_MOTOR = 11;
+    public static final int BACK_RIGHT_MODULE_STEER_ENCODER = 12;
+    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(-32.881 + 172.88 + 180.0 + 4.83);
+
+    // *****************************************************************
+    // standard stuff constants - motors rotation, etc.
+    public static final double DegreesPerRevolution = 360.0;
+    // NEO maximum RPM
+    public static final double neoMaximumRevolutionsPerMinute = 5676;
+    // NEO 550 maximum RPM - see:
+    // https://www.revrobotics.com/rev-21-1651/#:~:text=The%20following%20specifications%20for%20the%20NEO%20550%20Brushless,Motor%20Kv%3A%20917%20Kv%20Free%20Speed%3A%2011000%20RPM
+    public static final double neoFiveFiveZeroMaximumRevolutionsPerMinute = 11000;
+    // this uses the halls effect sensor when plugged into the spark max
+    // see: https://www.revrobotics.com/rev-21-1650/ where it says "42 counts per
+    // rev."
+    public static final double RevNeoEncoderTicksPerRevolution = 42;
+
+    // *****************************************************************
+    // input device constants
+    public static final int portDriverController = 0;
+    public static final int portCoDriverController = 1;
+
+    // *****************************************************************
+    // navx isLevel tolerence in degrees
+    public static final double navxTolDegrees = 4;
+    // ************************************
+    // trajectory constants
+    public static final double TrajectoryMaxAcceleration = 1;
+    public static final double TrajectoryMaxVelocity = 1;
+    // tolerence on trajectory locations
+    // TODO test tightening up these values
+    public static final Pose2d TrajectoryPoseTol = new Pose2d(0.1, 0.1, Rotation2d.fromDegrees(5));
+
+    // ***************************************
+    // For auto constants
+    public static final double snoutDepth = Units.inchesToMeters(2.75);
```

Comment:
This doesn't exist on this robot as best as I'm aware

Reply:
It is actually used in some auto routines. I've left a TODO in AutonoumousChooser to clean this up. Josie said this is to big to be included in this update.